### ISSUE TITLE
Harder to setup

### ISSUE THREAD

**Issue by kit-johnson** [rupor-github/wsl-ssh-agent#11]
As part of the description for this project, it says it aims to be "much easier to use" than the original ssh-agent-wsl. However I found the instructions in ssh-agent-wsl much more simple to follow: there was a 1, 2, 3 process for setting up which didn't require any choices from the user, and it worked perfectly for me.

In wsl-ssh-agent, there are several options to choose from in step 2 of "Usage", and it was very unclear to me which one I should be choosing. After reading them several times I took a stab at the first option, and eventually it worked. It would be really helpful to call this the 'standard' or 'default' option, for people like me who don't have a clue which one to choose.

**Comment by rupor-github** [comment]
Instructions in "old" ssh-agent-wsl did not explain how to manage agent life time at all, implicitly relying on the fact that people know how to manage ssh-agent under linux/unix. This assumption is quite often incorrect. Combined with WSL related issues leads to some shell code which is ugly at best (this I guess was the reason why it was left out :)). You could see some of the suggestions in the issues, for example here: https://github.com/rupor-github/ssh-agent-wsl/issues/7. 

"New" wsl-ssh-agent-gui aims to avoid this. It does not matter which way you'd choose in step 2 - all of them should work, however there is an example at the end of readme which you could use to repeat my setup (it is not necessary to take everything, however I show what I use there quite clearly I think).

**Comment by kit-johnson** [comment]
Thanks for the reply. Perhaps it's just the wording that confused me: "Basically there are several possible scenarios". To me that feels like I need to understand which scenario is the correct one for me, but I couldn't, and it turns out they would all be fine for me to use.

Maybe rewording it to "There are several ways to to this:"

**Comment by rupor-github** [comment]
As you can possibly guess English is not my first language - I will use your suggestion. thank you.

**Comment by kit-johnson** [comment]
> As you can possibly guess English is not my first language

I hadn't noticed. Your English is excellent!
